src/qml/qgismobileapp.qml

```diff
@@ -1028,6 +1043,10 @@ ApplicationWindow {
     }
   }
 
+  MapCanvasPointHandler {
+    id: mapCanvasPointHandler
+  }
```

Comment:
Oh and, we need have to sit as a child of Item { id: mapCanvas ... } so people can easily reach it like this:

iface.mapCanvas().pointHandler.registerHandler(...)

(and yes, relocated there I'd rename its id from mapCanvasPointHandler to pointHandler)